src/baldr/graphreader.cc

```diff
@@ -305,6 +305,7 @@ bool GraphReader::AreEdgesConnected(const GraphId& edge1, const GraphId& edge2)
     } else {
       const GraphTile* tile = GetGraphTile(n1);
       const NodeInfo* ni = tile->node(n1);
+      if (ni->transition_count() == 0) return false;
```

Comment:
this node only exists on this level so we cant expand

Reply:
correct.  No transitions from this node